recipe/meta.yaml

```diff
@@ -27,7 +27,6 @@ requirements:
     - pip
     - poetry-core >=1.0.0a9
     - python
-    - setuptools
     - wheel
   run:
     - python >=3.6
```

Comment:
If we skip `py<36` https://github.com/AnacondaRecipes/tomlkit-feedstock/pull/9/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR14 then we can use a plain `python` without pinning in `run`.